PROJECT_MANAGEMENT.md

```diff
@@ -0,0 +1,83 @@
+Here you will find a guide about how we project manage on github. This guide
+is intended to make it easier for you to find tasks to get involved with and
+how we manage work.
+
+We use Github issues to create, discuss or carry out tasks. Each issue
+can take one of three forms, a either a bug, a feature request or feature
+implementation (task). 
+
+- Bug: a problem with the code that needs fixing 
+- Feature request: an idea for new features 
+- Feature implementation: details and suggested steps to be completed to add
+ the feature/fix the bug
+
+Bug and feature requests become feature implementations when someone has a
```

Comment:
Same comment as above regarding this topic: advise to remove "Bug and". But also this is duplicate info. Which would be hard to manage. Maybe use this file as the source of truth? Not sure,

Reply:
I will delete this from the PROJECT_MANAGEMENT file which will contain only general information about the team project boards.